### PR TITLE
warn when using nested ternary

### PR DESCRIPTION
**Pull request by dwheale**

### CODE REVIEW

**Comment by dwheale** [comment]
This will suggest engineers write code that is easier to read and more maintainable.

File: packages/eslint/core.js [hunk 1]
```diff
@@ -60,8 +60,8 @@ module.exports = {
     ],
 
     // who doesn't love a good nested ternary?
```
Comment (chadly) [inline packages/eslint/core.js:62]: you have to change the snarky comment, too 😉

File: packages/eslint/core.js [hunk 2]
```diff
@@ -60,8 +60,8 @@ module.exports = {
...
-    'no-nested-ternary': 'off',
-    'unicorn/no-nested-ternary': 'off',
+    'no-nested-ternary': 'warn',
+    'unicorn/no-nested-ternary': 'warn',
```
Comment (chadly) [inline packages/eslint/core.js:64]: I'm more of a type-a personality. Should we go "all the way" with `error` instead?

**Review by chadly — CHANGES_REQUESTED** [review]

File: packages/eslint/core.js (hunk 2 above)
Comment (chadly) [reply packages/eslint/core.js:64]: nah, after sleeping on it, it seems better as a "suggestion" / warning like you did here

File: packages/eslint/core.js (hunk 1 above)
Comment (chadly) [reply packages/eslint/core.js:62]: actually, I think the comment still makes sense since it's transitioned to warn, not error

**Review by chadly — APPROVED** [review]

**Pull request merged**